dependencies.yaml

```diff
@@ -34,13 +34,11 @@ cover_requires:
 
 devel_requires:
   '%test_requires':
-  '%requires_not_needed_in_tests':
```

Comment:
why change that as well?

Reply:
as the commit message says: "so we can...remove the concept of %requires_not_needed_in_tests entirely as git was the last thing in it."

Reply:
I see. this key was only needed in spec and only for git. Fine